src/main/java/com/linecorp/armeria/server/thrift/xml/ThriftXmlDocString.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.thrift.xml;
```

Comment:
Give than we have hard dependency on Thrift types in `DocService` already, could we:

- Move `ThriftXmlDocString` to the same package with `DocService`, and
- make it and its public members package-local?